lib/WWW/Spotify.pm

```diff
@@ -159,6 +159,34 @@ has ua => (
     },
 );
 
+has 'response_status' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
+
+has 'response_content_type' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
```

Comment:
I wonder if rather than having these attributes, you could instead have a `_response` attribute which tracks the last response. You could then add two methods: `response_status` and `response_content_type` which would then use the `HTTP::Response` object to get this information.

Or, you could expose the response object itself and let the user check the status, content_type etc.

Reply:
There the response header is available via response_header, but that might present a bit more effort than people would like for what would be considered basic validation/sanity checking.